spec/controllers/users/phones_controller_spec.rb

```diff
@@ -16,6 +16,28 @@
       get :add
 
       expect(response).to render_template(:add)
+      expect(response.request.flash[:alert]).to be_nil
+    end
+
+    it 'displays error if phone number exceeds limit' do
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+      user.phone_configurations.create(encrypted_phone: '4105555555')
+
+      get :add
+      expect(response).to redirect_to(account_url + '#phones')
```

Comment:
Minor: Per the recommendation / change to use `anchor` keyword with the URL helper, should we do the same here?

```suggestion
      expect(response).to redirect_to(account_url(anchor: 'phones'))
```